.github/workflows/api-tests.yml

```
@@ -1,32 +1,29 @@
-name: Tests
-
+name: API Tests
```

```suggestion
name: NameGuard API Tests
```

I was going to name this `NameGuard API Tests`, but since this will only run when the NameGuard API folder is changed, I didn't really see the point. 

Feel free to commit this suggestion if we want it to be more explicit.

---

For me "API" is confusing. It is now referring to `api` directory which is the Python package.

---

@djstrong please suggest any changes. I've never looked at the files in those directories too close to know how things are structured.

---

Maybe just use the same string in filenames. Now we have `api` and `api-python`.
Also under namekit's `api` directory is NameGuard (Python). Maybe it should be under `api/nameguard`? If we plan to move more Python packages under `api` directory then it is probably a must. I guess Python packages can not be tested in one workflow then the names could be: 
* `api-python-release.yml` -> 'api-nameguard-release.yml'
* `api-tests.yml` -> `api-nameguard-tests.yml`
* `deploy-api.yml` -> `deploy-api-nameguard.yml`